src/main/java/fruitshop/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,20 @@
+package fruitshop.strategy;
+
+import fruitshop.db.Storage;
+import fruitshop.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage = new Storage();
+
+    @Override
+    public void handler(FruitTransaction fruitTransaction) {
+        int newQuantity = fruitTransaction.getQuantity()
+                + storage.getQuantity(fruitTransaction.getFruit());
+        if (newQuantity >= 0) {
+            storage.put(fruitTransaction.getFruit(), newQuantity);
```

Comment:
There is no need to check if `newQuantity` is greater than or equal to 0 in the `ReturnOperation`. The return operation should always increase the quantity of fruits in the storage. If you have specific business rules that require this check, please ensure they are documented.